Update the wording for screen reader rationale, add form template (#533)
govau/design-system-site: content/components/core/code/scss-functions.md

### AU-sronly

> Hide an element from the screen but not a screen reader. This mixin should be used to provide additional information or context to a webpage or form that the screen reader users would otherwise miss out on. With this mixin, the additional information will be picked up by a screen reader but will _not_ be displayed in the browser. 

Reviewer: ```suggestion
Hide an element from the screen but not a screen reader. This mixin should be used to provide additional information or context to an element on the page that screen reader users would otherwise miss out on. An example of this could be applyoing the mixin to  a `span` that has the text `close` on a button with a close icon. 
```

> An example of a use case for this class could be to provide more context to a `read more` link on a webpage.

Reviewer: ```suggestion
```

> Using this class, additional information about what the `read more` link relates to could be hidden from the browser, but will still be picked up by a screen reader. This would provide a screen reader user with the required context to understand what information is being provided.

Reviewer: ```suggestion
```
